README.md

```diff
@@ -1,8 +1,153 @@
 <!-- @format -->
 
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with
-[Create React App](https://github.com/facebook/create-react-app).
+# Project
 
-## Available Scripts
+> This Webpage is for a calculator app called Math
+> Magicians bad it was built using React!
+
+![screenshot]()
+
```

Comment:
- [ ] please make sure there is a screenshot of your application added here, now it is just an empty link

Reply:
Hi Reviewer, this is just one Milestone of the Math Magicians project and because the project is in build, I have not added the screenshots or live links yet. Once I'm done with the final Milestone with all the features added, then I'll add a proper screenshot of the working app and video links because right now the calculator is not even functioning, I hope that's fine

Reply:
Hi Zeeshan,
Regarding the screenshot, I do highly recommend adding one of the application how it looks like at the moment. You can update it as you progress. 
Regarding the live demo and the video demo, as I already mentioned you are free to not include them but make sure there are no empty links in your readme file.